src/frameworks/gatsby.json

```diff
@@ -23,6 +23,11 @@
     "AWS_LAMBDA_JS_RUNTIME": "nodejs14.x",
     "NODE_VERSION": "14"
   },
+  "logos": {
```

Comment:
Total nitpick: single `logo` instead of `logos` and then we could have `logo.default` rather than `logos.default`?

We do have a plural with `plugins` but that's an array and hence a list.

Reply:
I had done `logo` initially and then saw `plugins` and started doubting my choice 😂 Sounds good, I'll switch it back